spring-cloud-dataflow-server-local/docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '3'
```

Comment:
Suggestions:
- I'd like to also explore if we can auto-register the latest GA release of OOTB Stream and Task apps. It'd eliminate one more step from the [quick-start](http://cloud.spring.io/spring-cloud-dataflow/#quick-start).
- Since we have removed MySQL and Redis, it'd be good to document how to add it. 
- It'd be also good to document how to change DB or Kafka vs. Rabbit.
- We need to include the command (`docker exec -it springclouddataflow_dataflow-server_1 /bin/sh`) to SSH into the container for the app-logs. It'd be good to show an example (`log-sink` logs perhaps).

Reply:
fair points on the documentation bits, i held off on updating that until i got feedback so i can do it in one pass.

in terms of auto-registering, looks like thats available via https://docs.spring.io/spring-cloud-dataflow/docs/1.4.0.M1/reference/htmlsingle/#resources-app-registry-bulk -- just need to make sure the proper version is populated as with the comment about the image tag above

Reply:
Yes, we have the bulk-import API, and for the version, we will use the latest GA release of Apps. This is something we can manually change at the time of release as needed. 

Right now, they are at [Celsius.SR1](http://cloud.spring.io/spring-cloud-stream-app-starters/#stream-app-starters-and-spring-cloud-data-flow-) and [Clark.GA](http://cloud.spring.io/spring-cloud-task-app-starters/#task-app-starters-and-spring-cloud-data-flow-).